server/middleware/error_handler.js

```diff
@@ -1,11 +1,27 @@
 const _ = require('lodash');
-
+const curlify = require('request-as-curl');
 const errors = require('../lib/errors');
+const config = require('config');
+
+
+sendErrorByEmail = (req, err) => {
+  var errorHTML = 'To reproduce this error, run this CURL command:<br />\n<br />\n';
```

Comment:
Please use string templates, not concatenation (wonder why eslint doesn't complain)